<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManualCAD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/Resource/M.png</normaloff>:/icons/Resource/M.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>绘图引擎</string>
    </property>
    <addaction name="actionUseStupid"/>
    <addaction name="actionUseQtGui"/>
    <addaction name="actionUseOpenGL"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_read"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu1"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionPlaceLine"/>
   <addaction name="actionPlaceRect"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUseStupid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stupid模式</string>
   </property>
  </action>
  <action name="actionUseQtGui">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Qt内嵌模式</string>
   </property>
  </action>
  <action name="actionUseOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGL模式</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionPlaceLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/line.png</normaloff>:/icons/Resource/line.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="actionPlaceRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/rectangle.png</normaloff>:/icons/Resource/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/delete.png</normaloff>:/icons/Resource/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Resource/select.png</normaloff>:/icons/Resource/select.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
   <property name="toolTip">
    <string>选择</string>
   </property>
  </action>
  <action name="actionLineColor">
   <property name="text">
    <string>线条颜色</string>
   </property>
  </action>
  <action name="actionSelectedColor">
   <property name="text">
    <string>选中颜色</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="text">
    <string>背景色</string>
   </property>
  </action>
  <action name="actionhh">
   <property name="text">
    <string>电阻</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>二极管</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>三极管</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>电容</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>电感</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_read">
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
